docs/ja/feature_leader_key.md

```diff
@@ -0,0 +1,151 @@
+# リーダーキー: 新しい種類のモディファイア
+
+<!---
+  original document: 0.8.134:docs/feature_leader_key.md
+  git diff 0.8.134 HEAD -- docs/feature_leader_key.md | cat
+-->
+
+もしあなたが Vim を使ったことがある場合、リーダーキーは何であるかを知っています。そうでなければ、素晴らしい概念を発見しようとしています。:) 例えば、Alt+Shift+W を押す(3つのキーを同時に押す)代わりに、キーの_シーケンス_を押すことができたらどうでしょう？つまり、特別なモディファイア (リーダーキー)を押して、続けて W と C を押すと (単純にキーを高速に繋げます)、何かが起こります。
+
+それが `KC_LEAD` の機能です。以下は例です:
+
+1. リーダーキーとして使いたいキーボードのキーを選択します。それにキーコード `KC_LEAD` を割り当てます。このキーはこのためだけの専用です -- 単一アクションのキーで、他の用途には使うことができません。
+2. `config.h` 内に `#define LEADER_TIMEOUT 300` という行を追加します。これによって `KC_LEAD` キーのタイムアウトを設定します。具体的には、`KC_LEAD` キーを押してからリーダーキーのシーケンスを完了するまで一定の時間しかありません。ここでの `300` はそれを300msに設定します。この値を増やして、シーケンスを入力する時間を増やすことができます。ただし、この時間中に押されたキーは全て途中で遮られ、送信されません。そのためこの値は小さくしておいたほうが良いかもしれません。.
+   * デフォルトでは、このタイムアウトは、`KC_LEAD` を押してからシーケンス全体が完了するまでに掛かる時間です。これは一部の人にとっては非常に短いかもしれません。そのため、このタイムアウトを増やしたほうが良い場合もあります。必要に応じて、`LEADER_PER_KEY_TIMING` オプションを有効にしたほうが良い場合もあります。これは各キーがタップされる度にタイムアウトまでの時間をリセットする機能です。これにより、タイムアウト時間を短くしつつも、比較的長いシーケンスを使うことができます。このオプションを有効にするには、`config.h` に `#define LEADER_PER_KEY_TIMING` を追加します。
+3. `matrix_scan_user` 関数の中で、以下のようなものを追加します:
+
+```c
+LEADER_EXTERNS();
+
+void matrix_scan_user(void) {
+  LEADER_DICTIONARY() {
+    leading = false;
+    leader_end();
+
+    SEQ_ONE_KEY(KC_F) {
+      // マクロ内でできること
+      SEND_STRING("QMK is awesome.");
+    }
+    SEQ_TWO_KEYS(KC_D, KC_D) {
+      SEND_STRING(SS_LCTL("a") SS_LCTL("c"));
+    }
+    SEQ_THREE_KEYS(KC_D, KC_D, KC_S) {
+      SEND_STRING("https://start.duckduckgo.com\n");
+    }
+    SEQ_TWO_KEYS(KC_A, KC_S) {
+      register_code(KC_LGUI);
+      register_code(KC_S);
+      unregister_code(KC_S);
+      unregister_code(KC_LGUI);
+    }
+  }
+}
+```
+
+ご覧のとおり、幾つかの機能があります。`SEQ_ONE_KEY` を単一キーシーケンス (リーダーの後に1つのキーのみ)に使い、より長いシーケンスについては `SEQ_TWO_KEYS`、`SEQ_THREE_KEYS` から `SEQ_FIVE_KEYS` を使うことができます。
+
+これらはそれぞれ1つ以上のキーコードを引数として受け付けます。これは重要な点です: **キーボードの任意のレイヤー**のキーコードを使うことができます。当たり前ですが、リーダーマクロが発動するにはそのレイヤーがアクティブである必要があります
+
+## `rules.mk` にリーダーキーサポートを追加
+
+リーダーキーのサポートを追加するには、単純にキーマップの `rules.mk` に1行を追加します:
+
+```make
+LEADER_ENABLE = yes
+```
+
+## リーダーキーのキーごとのタイミング
+
+長いリーダーキー文字列のために長いタイムアウト時間を設定したくない場合や、200wpm のタイピングスキルが無い場合、キーを押すごとに入力を完了するまでの時間を増やす機能を使用することができます。これは、リーダーキーを使ってタップダンスを再現する場合に非常に役立ちます (C, C, C のような同じキーを複数回タップする場合)。
```

Comment:
「Rather 〜 for long leader key strings」の中に「したくない」という否定を表す単語が無いので、次の訳はどうでしょうか。
```suggestion
長いリーダーキー文字列のために非常に長いタイムアウト時間に頼ったり、200wpm のタイピングスキルが無い場合、キーを押すごとに入力を完了するまでの時間を増やす機能を使用することができます。これは、リーダーキーを使ってタップダンスを再現する場合に非常に役立ちます (C, C, C のような同じキーを複数回タップする場合)。
```

Reply:
ご指摘ありがとうございます。元の訳文の回りくどさがなくなった気がします。

Reply:
Rather than relying on an incredibly high timeout （略）we can ...は、「非常に長いタイムアウト時間に頼りたくない場合は代わりに...」という意味なので、「長いリーダーキー文字列のために非常に長いタイムアウト時間に頼ったり（略）キーを押すごとに入力を完了するまでの時間を増やす機能を使用することができます。」では、前半部の代替手段として後半部があるというのが伝わりにくいと考えています。
（また「たり」が1つしかないのも日本語の一般的な用法から外れます）
変更前の和訳は、元の文におけるan incredibly high timeout for long leader key stringsとthose of us without 200wpm typing skillsの並列を直訳しても日本語としては意味が非常に通じにくいので、多少意訳になっても日本語として意味が伝わりやすい文にすべきとの考えで作成しました。

Reply:
> Rather than relying on an incredibly high timeout （略）we can ...は、「非常に長いタイムアウト時間に頼りたくない場合は代わりに...」という意味なので、「長いリーダーキー文字列のために非常に長いタイムアウト時間に頼ったり（略）キーを押すごとに入力を完了するまでの時間を増やす機能を使用することができます。」では、前半部の代替手段として後半部があるというのが伝わりにくいと考えています。

この点はご指摘通りで、私も悩んだ末に強引に上記のとおり訳したところです。それでも、依然として後半部の存在意義が伝わりにくいので、あまり役に立たない提案をしたことになってしまいました。

原文通りに訳し、かつ、日本語として通じる文章にするのは相当に難しそうなので、意訳になるのは仕方ないと考えて、

「長いリーダーキー文字列のために非常に長いタイムアウト時間に頼る、または200wpm のタイピングスキルが無い場合、代わりにキーを押すごとに入力を完了するまでの時間を増やす機能を使うことができます。」

という具合に、「または」と「代わりに」の2単語を追加するのはどうでしょうか？

Reply:
並列なので「頼る」＋「～ない場合」とするより、「頼りたくない場合」＋「～ない場合」と揃えた方が日本語としては自然な感じになると思います。

「長いリーダーキー文字列のために非常に長いタイムアウト時間に頼りたくない場合、または200wpm のタイピングスキルが無い場合、代わりにキーを押すごとに入力を完了するまでの時間を増やす機能を使うことができます。」

Reply:
確かに、語尾を揃えるほうが自然な感じになりますね。
「without 200wpm typing skills」は「200wpm のタイピングスキルが無い場合」という訳になりますから、「Rather than 〜 strings」に否定系の単語はないですが「スキルが無い場合」に合わせる形で「頼りたくない場合」と訳すのは妥当だと思います。